src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.cs

```diff
@@ -578,6 +578,29 @@ protected override ExpressionSyntax CreateCallSignature()
                     return invocation;
                 }
 
+                if (this.CSharpSelectionResult.ShouldCallConfigureAwaitFalse())
+                {
+                    if (this.AnalyzerResult.ReturnType.GetMembers().Any(x => x is IMethodSymbol
+                    {
+                        Name: "ConfigureAwait",
+                        Parameters:
+                        {
+                            Length: 1
+                        } parameters
```

Comment:
let's play around with this a bit.  I think it would be good to have a balance of horizotnal and vertical matching.  How about:

```c#
 x is IMethodSymbol
{
  Name: "ConfigureAwait",
  Parameters: {  Length: 1 } parameters
} &&
parameters[0].Type is { SpecialType: SpecialType.System_Boolean }
```

Basically, be willing to inline simple property patterns to not feel like a simple concept (a 1-length parameter array) needs so many lines of code to express.

Reply:
I tried doing this, but Vs have me warnings that it wasn't formatted correctly.